imaids/utils.py

```diff
@@ -5,7 +5,12 @@
 import radia as _rad
 
 
-mu0 = 4*_np.pi*1e-7
+# NOTE: package lnls-sirius/mathphys could be used to defined these consts
+
+ELEMENTARY_CHARGE = 1.60217662e-19  # [C]
+ELECTRON_MASS = 9.10938356e-31  # [Kg]
+LIGHT_SPEED = 299792458  # [m/s]
+VACUUM_PERMEABILITY = 1.25663706212e-6  # [V.s/A/m]
```

Comment:
The package already have a get_constants() function in the utils model that returns the electron mass and speed of light. Such method is used in certain parts of the code, such as in trajectory calculations. I don't think it might be a good ideal to superimpose these values here

However, I would agree that the get_constants() approach has its issues. Mainly, the function also returns all constants in every case at the same order. We cannot call a single function nor changing their order in the function without breaking other parts of the code.

I would like to propose that we deal with this issue in a dedicated patch, not this one. Standardizing the constants definition in all parts of the code.
A possibility would be just calling the scipy.constants module (https://docs.scipy.org/doc/scipy/reference/constants.html) which have all the necessary values up to date with the 2018 SI definition.